### PR TITLE
Support custom endpoint

### PR DESCRIPTION
**Pull request by d-kuro**
# Useage

```sh
$ docui -h
Usage of docui:
  -ca string
        ca.pem file path
  -cert string
        cert.pem file path
  -endpoint string
        Docker endpoint (default "unix:///var/run/docker.sock")
  -key string
        key.pem file path
```

Or set environment variable:

* `DOCKER_HOST`
* `DOCKER_TLS_VERIFY`
* `DOCKER_CERT_PATH`

## Support protocol

* [x] unix
* [x] tcp
* [x] http
* [x] https
* [ ] ssh 
  * non support `fsouza/go-dockerclient`
  * [client.go#L991](https://github.com/fsouza/go-dockerclient/blob/cb5c4abcfae0a9effa0c1390907389f090b6e2f4/client.go#L991)
  * **Non verifyed:** Environment variable `DOCKER_HOST`

## Sample

```sh
$ docui --endpoint "tcp://54.250.44.211:2375"
```

### CODE REVIEW

File: wiki.md [hunk 1]
```diff
@@ -223,3 +223,30 @@ If you want to specify multiple options, please enter as below.
...
 type=nfs o=addr=192.168.1.1,rw device=:/path/to/dir
 ```
+
+## Configuration
+
+### Command-Line Options
+
+Support custom endpoint:
+
+```text
```
Comment (skanehira) [inline wiki.md]: Please use `shell`.

Comment (skanehira) [inline wiki.md]: Please use `shell`

Comment (d-kuro) [reply wiki.md]: Fixes: https://github.com/skanehira/docui/pull/70/commits/0e222e8f4b3da543253dea5e1a4cffba087819cd

Comment (skanehira) [reply wiki.md]: Thanks!

**Comment by skanehira** [comment]
Thank you!

**Pull request merged**